sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/ExceptionEventData.java

```diff
@@ -18,13 +17,15 @@
 public interface ExceptionEventData extends EventData {
 
   /**
-   * Returns a new immutable {@link ExceptionEventData}.
+   * Returns a new immutable {@link io.opentelemetry.sdk.trace.data.ExceptionEventData}.
```

Comment:
this shouldn't require being fully qualified, I don't think. I'd revert the fully-qualified classname changes here.

Reply:
Dang it, sorry I will just use the class name and remove the fully-qualified classname.